android/engine/src/main/java/org/smartregister/fhircore/engine/util/HelperConstants.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2021 Ona Systems, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.smartregister.fhircore.engine.util
+
+const val TRUE = "true"
```

Comment:
why are these 3 needed?

Reply:
it was added as per Ben's review, later handled with Boolean, 
though its removed now,  and required cursorSymbol is moved under PinView class